src/createContext.ts

```diff
@@ -98,9 +106,9 @@ export const importBuiltins = (context: Context, externalBuiltIns: CustomBuiltIn
       if (objs.hasOwnProperty(i)) {
         const val = objs[i]
         if (typeof Math[val] === 'function') {
-          defineSymbol(context, 'math_' + val, Math[val].bind())
+          defineBuiltin(context, 'math_' + val, Math[val].bind())
```

Comment:
I believe the `.bind` is useless. Also can simplify if there is only one way to define a builtin.